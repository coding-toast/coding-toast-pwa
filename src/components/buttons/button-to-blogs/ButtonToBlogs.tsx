import * as React from 'react'
import ButtonToBlogsStyles from './ButtonToBlogs.module.scss'
import Button from '../Button'

const ButtonToBlogs: React.FC = () => {
  return <Button toUrl="/" buttonText="Go To All Posts" className={`${ButtonToBlogsStyles.primaryBtn} text-style-label`} />
}

export default ButtonToBlogs
