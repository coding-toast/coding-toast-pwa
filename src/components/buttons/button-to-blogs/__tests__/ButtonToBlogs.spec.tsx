import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ButtonToBlogs from '../ButtonToBlogs'

describe('<ButtonToBlogs />', () => {
  it('renders correctly', () => {
    const component = shallow(<ButtonToBlogs />)
    expect(toJson(component)).toMatchSnapshot()
  })
})
