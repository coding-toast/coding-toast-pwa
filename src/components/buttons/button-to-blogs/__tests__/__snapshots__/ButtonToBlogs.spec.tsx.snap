// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonToBlogs /> renders correctly 1`] = `
<Button
  buttonText="Go To All Posts"
  className="primaryBtn text-style-label"
  toUrl="/"
/>
`;
